### ISSUE TITLE
No field `holderName` at RecurringDetail

### ISSUE THREAD

**Issue by Kolyall** [Adyen/adyen-android#119]
add pls field `holderName` to model `com.adyen.checkout.base.model.paymentmethods.RecurringDetail`

```
───── Response ───────────────────────────────────────────────────────────────────────
URL: https://checkout-test.adyen.com/v49/paymentMethods

{
...
"storedPaymentMethods": [
   {
      "brand": "visa",
      "expiryMonth": "10",
      "expiryYear": "2020",
      "holderName": "re",
      "id": "8415710463615192",
      "lastFour": "1111",
      "name": "VISA",
      "supportedShopperInteractions": [
         "Ecommerce",
         "ContAuth"
      ],
      "type": "scheme"
   }
]

...

}
```

**Comment by Kolyall** [comment]
also rename `id` -> `recurringDetailReference`
because it has the name on response of 
```
URL: https://checkout-test.adyen.com/v49/payments

Body:
{
   "additionalData": {
      "recurringProcessingModel": "CardOnFile",
      "recurring.recurringDetailReference": "8415710463615192",
      "recurring.shopperReference": "shopperReferenceId"
   },
   "pspReference": "852571066304314D",
   "resultCode": "Authorised",
   "merchantReference": "android-test-components"
}
```

**Comment by caiofaustino** [comment]
Thanks for the catch with the `holderName`. Also there was some confusion on our side, since the API response changed we updates the content of the object but forgot to update the Class name.

Next version I'm renaming `RecurringDetail` to `StoredPaymentMethod`.

`RecurringDetail` is the object inside the `oneClickPaymentMethods` but we are actually using the array from the `storedPaymentMethods` object which is type `StoredPaymentMethod`
Reference [Docs](https://docs.adyen.com/api-explorer/#/PaymentSetupAndVerificationService/v50/paymentMethods__section_resParams)

Will be available on the next release, sorry for the confusion.

**Comment by caiofaustino** [comment]
This is released in `3.5.0`, can you check if it solves the issue for you?

**Comment by caiofaustino** [comment]
If you still find issues just let us know.
